manim/mobject/types/vectorized_mobject.py

```diff
@@ -433,11 +435,28 @@ def set_points(self, points):
     def get_points(self):
         return np.array(self.points)
 
-    def set_anchors_and_handles(self, anchors1, handles1, handles2, anchors2):
+    def set_anchors_and_handles(
+        self,
+        anchors1: Iterable[float],
+        handles1: Iterable[float],
+        handles2: Iterable[float],
+        anchors2: Iterable[float],
+    ) -> "VMobject":
+        """Given two sets of anchors and handles, process them to set them as anchors and handles of the VMobject. anchors1[i], handles1[i], handles2[i] and anchors2[i] define the
+        i-th bezier curve of the vmobject.
+
+        There are four hardcoded paramaters and this is a problem as it makes the number of points per cubic curve unchangeable from 4. (two anchors and two handles).
+
+        Returns
+        -------
+        VMobject
+            for chaining.
+        """
```

Comment:
Nitpick: review https://numpydoc.readthedocs.io/en/latest/format.html#sections, specifically the part about the "short summary": "A one-line summary that does not use variable names or the function name". If the docstring needs to refer to variable names, it is recommended to do that in the notes section or in the extended summary part.

Reply:
I didn't know that. thanks !